src/code/z_vimode.c

```diff
@@ -158,7 +158,7 @@ void ViMode_Configure(OSViMode* viMode, s32 type, s32 tvType, s32 loRes, s32 ant
     } else {
         viMode->fldRegs[0].vStart += START((u16)-3, (u16)-2);
         if (tvType == OS_TV_MPAL) {
-            viMode->fldRegs[0].vBurst += BURST((u8)-2, (u8)-1, 12, -1);
+            viMode->fldRegs[0].vBurst += BURST((u8)-2, (u8)-1, 12, (u8)-1);
```

Comment:
If these macros are ours it seems like it'd be better to put the casts in the macro itself? Or make a wrapper with casts if they aren't ours.

Reply:
I consulted the libultra experts and they said it is better to move the casts inside the macros, so that's what I did